python/activity_stream/widget_activity_stream_base.py

```diff
@@ -115,20 +115,21 @@ def _set_timestamp(self, data, label):
         
         # standard format 
         full_time_str = datetime_obj.strftime('%a %d %b %Y %H:%M') 
-    
-        if datetime_obj > datetime.datetime.now():
-            # future times are reported precisely
-            return (full_time_str, full_time_str) 
-        
+
+        time_now_obj = datetime.datetime.now()
+        # The note created time is rounded up to the nearest second by Shotgun,
+        # so handle creation time greater than now.
```

Comment:
I'm not sure this is true in all cases, but it was observed to be true for "Reply" entity types. Note Reply creation dates can be rounded to the next minute. I'm not sure if this happens all the time and it depends on how long it takes for the panel to refresh, or if it only happens sometimes, since this bug was not visible all the time for us.